### PR TITLE
Assign FormattedText._platform in constructor #5576

### LINKED ISSUE #5576: FormattedText constructor with 6 params (string, Typeface, ...) doesn't assign _platform field. NullReferenceException when rendering such FormattedText.
**Describe the bug**
FormattedText constructor with 6 params (string, Typeface, ...) doesn't assign _platform field. NullReferenceException when rendering such FormattedText.

### PR DESCRIPTION
**Pull request by antrv**
## What does the pull request do?
This PR fixes NullReferenceException when rendering FormattedText, created using constructor with 6 parameters. The _platform field was unassigned in this constructor.

## What is the updated/expected behavior with this PR?
No NRE when rendering FormattedText created using any constructor.

## Checklist
- [ ] Added unit tests (if possible)? No new tests.
- [ ] Added XML documentation to any related classes? 
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Avalonia with user documentation

## Breaking changes
None.

## Fixed issues
Fixes #5576

### CODE REVIEW

**Comment by Gillibald** [comment]
This is by design
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Visuals/Media/FormattedText.cs#L143

There is nothing to fix here

**Comment by antrv** [comment]
> This is by design
> https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Visuals/Media/FormattedText.cs#L143
> 
> There is nothing to fix here

I don't quite understand. The `_platformImpl` is created using `_platform`, which is null, and I get NRE.

File: src/Avalonia.Visuals/Media/FormattedText.cs [hunk 1]
```diff
@@ -47,6 +47,8 @@ public FormattedText(IPlatformRenderInterface platform)
         public FormattedText(string text, Typeface typeface, double fontSize, TextAlignment textAlignment,
             TextWrapping textWrapping, Size constraint)
```
Comment (Gillibald) [inline src/Avalonia.Visuals/Media/FormattedText.cs:48]:
```suggestion
            TextWrapping textWrapping, Size constraint) : this()
```

**Comment by Gillibald** [comment]
Y you are right. I prefer calling the parameterless ctor in the overload.

File: src/Avalonia.Visuals/Media/FormattedText.cs (hunk 1 above)
Comment (antrv) [reply src/Avalonia.Visuals/Media/FormattedText.cs]: I did the suggested changes.

**Review by Gillibald — APPROVED** [review]

**Pull request merged**